README.md

```diff
@@ -129,12 +129,25 @@ You can also specify build metadata in a version tag. If the tag is on the curre
 
 ### Can I use the version calculated by MinVer for other purposes?
 
-Yes! MinVer sets both the `Version` and `PackageVersion` MSBuild properties. Use them in a target which runs after MinVer. E.g.
+Yes! MinVer sets the `Version`, `PackageVersion`, `AssemblyVersion`, `MinVerMajor`, `MinVerMinor`, and `MinVerPatch` MSBuild properties. You can use them, or override their values, in a target which runs after MinVer.
+
+For example, MinVer sets `AssemblyVersion` to `{major}.0.0.0`, as recommended in the official [open source library guidance](https://docs.microsoft.com/en-ca/dotnet/standard/library-guidance/versioning#assembly-version). For projects which do not create NuGet packages, you may want to override this behaviour and populate [all four parts](https://docs.microsoft.com/en-us/dotnet/framework/app-domains/assembly-versioning#assembly-version-number) of `AssemblyVersion`. E.g. using Appveyor:

```

Comment:
You use `{major}.0.0.0` here, but `{MAJOR}.0.0.0` above.

Reply:
👍 I'll change it to upper case here.